### PR TITLE
SupportTicket -> ErrorReport

### PR DESCRIPTION
**Pull request by vanwagonet**
This was the last real thing inside keymaster.

[ignore-commit-lint]

### CODE REVIEW

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 95.06% | 94.9% | 0.16%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/ErrorReports/APIErrorReport.swift [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        let category: String?
+        let code: Int?
+        let comments: String?
+        let description: String?
+        let email: String?
+        let http_env: [String: String]?
+        let message: String?
+        let subject: String
+        let url: URL?
+        let user_percieved_severity: Severity?
```
Comment (armstrongnate) [inline Core/Core/ErrorReports/APIErrorReport.swift]: typo: `user_perceived_severity`

File: Core/Core/Views/ItemPickerViewController.swift [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+public class ItemPickerCell: UITableViewCell {
+    @IBOutlet weak var titleLabel: UILabel?
+    @IBOutlet weak var subtitleLabel: UILabel?
+}
+
+public protocol ItemPickerDelegate: class {
+    func itemPicker(_ itemPicker: ItemPickerViewController, didSelectRowAt indexPath: IndexPath)
+}
+
+public class ItemPickerViewController: UITableViewController {
```
Comment (armstrongnate) [inline Core/Core/Views/ItemPickerViewController.swift]: This is nice!

**Comment by armstrongnate** [comment]
Looks like we need to quarantine `AssignmentPostPolicyTests.testPostPolicySettings()` with this.

**Pull request merged**